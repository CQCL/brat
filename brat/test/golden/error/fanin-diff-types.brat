f(Qubit, Bit) -o Vec(Qubit, 2)
f = { [\/] }
