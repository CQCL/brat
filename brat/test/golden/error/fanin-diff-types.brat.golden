Error in test/golden/error/fanin-diff-types.brat@2:5
f = { [\/] }
    ^^^^^^^^

  Type mismatch when checking [\/]
Expected: Qubit
But got:  Bit


